FinSearchUnified/Resources/config.xml

```diff
@@ -27,6 +27,35 @@
             <description lang="de">Aktiviert die FINDOLOGIC Suche für Kategorieseiten.</description>
             <description lang="en">Activate the FINDOLOGIC search provider for category pages.</description>
         </element>
+        <element type="combo">
+            <name>CrossSellingCategories</name>
+            <label lang="de">Cross-Selling Kategorien (Produkte in diesen Kategorien werden vom Export ausgeschlossen)</label>
+            <label lang="en">Cross-Selling categories (products in these categories are excluded from the
+                export)</label>
+            <description lang="de">Sollten Sie mehrere Subkategorien in Ihrer Cross-Selling Kategorie haben, fügen sie jede Subkategorie ein.</description>
+            <description lang="en">If you have multiple subcategories in your Cross-Selling category, make sure to add all subcategories.</description>
+            <store><![CDATA[Ext.define('Findologic.Category.Form', {
+    extend: 'Shopware.apps.Base.store.Category',
+    autoLoad: true,
+    pageSize: 25,
+    proxy: {
+        type: 'ajax',
+        url: window.location.href.substr(0, window.location.href.indexOf('backend')) + 'backend/Category/getPathByQuery',
+        reader: {
+            type: 'json',
+            root: 'data',
+            totalProperty: 'total'
+        }
+    }
+    }).create();//new ]]>
```

Comment:
I do not think that this comment `//new` is necessary.

Reply:
It is mentioned in their documentation that it is a sort of hack to load the store.

Reply:
I see. If it works, then ok :).